tools/test-proxy/Azure.Sdk.Tools.TestProxy.Tests/Test.Certificates/test_pem_key

```diff
@@ -0,0 +1,24 @@
+﻿-----BEGIN PRIVATE KEY-----
```

Comment:
These are from a long-expired cert value/key pair. Not a security issue, nor is the test_public-key-only_pem. That is generated as indicated in `about.md`